desktop-environment.el

```diff
@@ -164,6 +164,23 @@ portion of the screen."
   :type 'string)
 
 
+
+;;; Customization - keyboard layout
+
+(defcustom desktop-environment-keyboard-layouts '("us" "us -variant dvorak" "cz")
```

Comment:
I don't like having a default value that only works for you :-). If the current configuration can be retrieved, is it possible that the default value is a list of just one item which is the current configuration? Otherwise, an empty list might be good enough.

If you do this change, please move your configuration to the docstring so the user gets a good example to start with.

Reply:
It was only as an example but you are right, having it in docstring is much better idea.

> is it possible that the default value is a list of just one item which is the current configuration?

I am not sure if xkeyboard supports retrieving current X layout. Also I would have to make sure to overwrite it with user's own list once he changes it.

Reply:
> I would have to make sure to overwrite it with user's own list once he changes it.

That's automatic. A default value on a `defcustom` is overriden by a value that a user specifies (even if the user specified the value before the `defcustom` expression was evaluated).